script/backfill-linked-accounts.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO linked_accounts (user_id, provider, account_id, access_token, login, avatar)
```

Comment:
We'll likely want a benchmark on this to see if it takes long enough that it needs to be broken into smaller batches, but my instinct says this approach is _probably_ fine.